llvm/lib/CAS/InMemoryCAS.cpp

```
@@ -228,6 +228,8 @@ class InMemoryCAS : public BuiltinCAS {
 
   void print(raw_ostream &OS) const final;
 
+  Error validate() const final { return Error::success(); }
```

Maybe return an error saying this is unimplemented?